class/defaults.yml

```diff
@@ -1,4 +1,14 @@
 parameters:
   appcat_service_postgresql:
-    =_metadata: {}
     namespace: syn-appcat-service-postgresql
+    charts:
+      appcat-service-postgresql:
+        source: https://charts.appuio.ch
+        version: 0.1.0
+    images:
+      appcat-service-postgresql:
+        registry: ghcr.io
+        repository: appcat/service-postgresql
+        tag: v0.1.0
```

Comment:
I don't think we need to maintain the image separately from the chart. Every time we release a new operator version, we should update the helm chart, which contains the new image tag.
So I think we can avoid this, otherwise Renovate creates 2 PRs for the same update. One for the chart version (which contains the updated image tag) and another for the image tag, which sets the image tag to the same value as already the default.
```suggestion
```